docs(object-storage): Adding how-to to restore all files from GLACIER in a bucket (#1017)
scaleway/docs-content: storage/object/how-to/restore-an-object-from-glacier.mdx

      Your object remains available in `STANDARD` class for the duration you chose during the restore process. It will be transferred automatically back to `GLACIER` once the configured period is over. 
    </Message>

## How to restore all objects in a bucket

> If you have numerous files in a bucket that you would like to restore, it may be more convenient to use the command line, for example with the s3cmd tool. To restore all files recursively in a bucket, you can invoke this command:

Reviewer: ```suggestion
If you have numerous files in a bucket that you would like to restore, it may be more convenient to use the command line, for example with the [s3cmd tool](/docs/tutorials/s3cmd/). To restore all files recursively in a bucket, you can invoke this command:
```

```
s3cmd restore -v --recursive s3://<bucket-name>
```

> To count the number of files still in GLACIER (see section on Restore time below, it is not immediate!), you can use:

Reviewer: ```suggestion
To count the number of files still in GLACIER (see [restore time](#restore-time) below, this is not immediate), you can use:
```
